crates/re_renderer/src/point_cloud_builder.rs

```diff
@@ -150,13 +153,14 @@ where
     /// Each time we `add_points`, or upon builder drop, make sure that we
     /// fill in any additional colors and user-data to have matched vectors.
     fn extend_defaults(&mut self) {
-        if self.0.colors.len() < self.0.vertices.len() {
-            self.0.colors.extend(
-                std::iter::repeat(Color32::WHITE).take(self.0.vertices.len() - self.0.colors.len()),
+        if self.0.color_buffer.num_written() < self.0.vertices.len() {
+            self.0.color_buffer.extend(
+                std::iter::repeat(Color32::WHITE)
+                    .take(self.0.vertices.len() - self.0.color_buffer.num_written()),
             );
         }
 
-        if self.0.user_data.len() < self.0.user_data.len() {
+        if self.0.user_data.len() < self.0.vertices.len() {
```

Comment:
nice save :trollface:

Reply:
that caused me some reeeaaal weird crash :D